acurl/acurl/__init__.py

```diff
@@ -379,7 +381,10 @@ def json(self):
     @property
     def headers(self):
         if not hasattr(self, '_headers'):
-            self._headers = dict(self.headers_tuple)
+            self._headers = defaultdict(list)
```

Comment:
I'd probably be tempted to use an intermediate variable here rather than use self._headers and then rewrite it on line 387. I was a little confused at first glance.